src/signTxOutput.h

```diff
@@ -54,7 +54,7 @@ typedef struct {
 	security_policy_t addressSecurityPolicy;
 
 	union {
-		basic_output_data_t output;
+		top_level_output_data_t output;
 		token_group_t tokenGroup;
```

Comment:
probably out of scope of this PR but to maintain consistency with developer docs and Trezor terminology, this should be named assetGroup/asset_group_t and token_amount_t should be token_t